jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -324,23 +326,61 @@ def plot_selected_points(self):
                                                            getattr(y, 'value', y))
 
     def vue_zoom_in(self, *args, **kwargs):
-        """This function will zoom into the image based on the selected points"""
+
+        self.zoom_to_selected(return_bounding_box=True)
```

Comment:
doesn't really matter, but might as well not pass anything here since nothing is being done with the returned bounding box